mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -204,12 +204,14 @@ function getAlbum (pl) {
 function artistsToString (artists, max) {
     var result = ''
     max = max || 3
-    for (var i = 0; i < artists.length && i < max; i++) {
-        if (artists[i].name) {
-            if (i > 0) {
-                result += ', '
+    if (artists && artists.length > 0) {
```

Comment:
Yes you can call ``join()`` on an array and specify the separator. I think we can also drop the ``max`` argument as it is not currently used anywhere. I'm hoping that the frontend HTML has been written in a way that will add ellipses if the text becomes too long.